frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -54,25 +54,25 @@ class WorkspaceAdvanced extends React.Component {
     addAllResourceI18n(i18n, this.state.config.translation, this.state.loggedUser.lang)
     i18n.changeLanguage(this.state.loggedUser.lang)
 
-    document.addEventListener('appCustomEvent', this.customEventReducer)
+    document.addEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT_LISTENER, this.customEventReducer)
   }
 
   customEventReducer = ({ detail: { type, data } }) => { // action: { type: '', data: {} }
     switch (type) {
-      case 'workspace_advanced_showApp':
+      case CUSTOM_EVENT.SHOW_APP(this.state.config.slug):
```

Comment:
please factorize with a `const { state } = this` and use `stage.config.slug` in this function
It's only a small code cleaning but it increase consistency between apps